sqlbrite/src/main/java/com/squareup/sqlbrite/BriteDatabase.java

```diff
@@ -149,8 +149,7 @@ SQLiteDatabase getReadableDatabase() {
     return db;
   }
 
-  // Package-private to avoid synthetic accessor method for 'transaction' instance.
```

Comment:
can you include the comments for `getReadableDatabase/getWritableDatabase` off `SQLiteDatabase` if there are any similar to what we do for the other copied apis